public/js/test/mochitest/browser_dbg-navigation.js

```diff
@@ -11,15 +11,13 @@ function countSources(dbg) {
  * navigating while paused will reset the pause state and sources
  */
 add_task(function* () {
-  const dbg = yield initDebugger(
-    "doc-script-switching.html",
-    "script-switching-01.js"
-  );
+  const dbg = yield initDebugger("doc-script-switching.html");
```

Comment:
You don't ever explicitly use `script-switching-01.js`, so you don't have to wait for it :)